lib/components/viewers/route-viewer.js

```diff
@@ -174,21 +174,23 @@ class RouteViewer extends Component {
             </div>
 
             <div className="header-text route-expanded">
-              {route && !route.pending && ModeIcon && (
-                <ModeIcon
-                  aria-label={getFormattedMode(
-                    getModeFromRoute(route).toLowerCase(),
-                    intl
-                  )}
-                  mode={getModeFromRoute(route)}
-                  width={22}
+              <h1 style={{ display: 'contents' }}>
+                {route && !route.pending && ModeIcon && (
```

Comment:
`route !== null` is already implied by the condition at Line 140.
```suggestion
                {!route.pending && ModeIcon && (
```